ktor-server/ktor-server-core/common/src/io/ktor/server/response/ApplicationResponseFunctions.kt

```diff
@@ -118,7 +118,7 @@ public suspend inline fun ApplicationCall.respondRedirect(permanent: Boolean = f
  *
  * @see [io.ktor.server.response.ApplicationResponse]
  * @param contentType is an optional [ContentType], default is [ContentType.Text.Plain]
- * @param status is an optional [HttpStatusCode], default is [HttpStatusCode.OK]
+ * @param status is an optional [HttpStatusCode], default is [HttpStatusCode.OK] unless already assigned
```

Comment:
For consistency with other KDocs, I suggest to omit the verb (is) in this and the next occurrences:

```suggestion
 * @param status An optional [HttpStatusCode], defaults to [HttpStatusCode.OK] unless already assigned
```